src/main/java/frc/robot/commands/DefaultDriveCommand.java

```diff
@@ -34,8 +34,7 @@ public void execute() {
 
         double forwardTranslation = driver.getForwardTranslation();
         double sideTranslation = driver.getSideTranslation();
-        // Negative because counterclockwise is positive for the robot and right is positive on the controller
-        double rotation = -driver.getRotation();
+        double rotation = swerve.getIsShooting() ? 0 : driver.getRotation();
```

Comment:
Don't use this part of the command for shooting. Update from main and the Vision branch and use the drive heading part below. There's a function in drivetrain, setIsAlign(), to set if you want to align to the speaker. Set that to true in the initialize of the shoot command and false at the end. If you update from main and vision, there should be another method, getAlignAngle(), in Drivetrain to return the angle to align to for scoring. Currently, this is the angle to the speaker, but if we want to shoot while moving, that needs to be changed to whatever calculation we use.

Reply:
i dont see the getAlignAngle() in the drivetrain class after updating from main. is it on the main branch?

Reply:
It was on the Vision branch. I merged that into this one, so you should see it now.

Reply:
moved the Shoot command into its own PR. we can continue there.